### ISSUE TITLE
SOD in call and SOD after pressing power button to lock screen

### ISSUE THREAD

**Issue by markob94** [CM-Cancro/bugreports#9]
latest testbuild cm-12.1-20150605-UNOFFICIAL-mi3-cancro
I got SOD in call after 30 sec ,and after that ,hard reboot and I take logcat..
https://drive.google.com/file/d/0Bz1uQMAZ-xvPcFlOU3R0X1k2UkU/view?usp=sharing

after that I press power button to lock screen, and after 1m I press volume button to wake screen,nothing,press power button, same.. I must hard reboot phone, there is logcat after reboot 
https://drive.google.com/file/d/0Bz1uQMAZ-xvPZTNhSDB4NF9GRm8/view?usp=sharing

**Comment by rooque** [comment]
Try to get logcat before hard reboot with adb  when SOD occur ... Logcat after reboot will say nothing about the bug. The only thing you can use to report an bug after rebooting is last_kmsg.

**Comment by markob94** [comment]
Yeah, I try that ,but my phone isn't recognized via adb, but in fastboot all is ok.. I try all ways but nothing ..

**Comment by kikorb** [comment]
Same thing here. Every time I have a phone call after 30 seconds SOD
